js/nested-for-loops.js

```diff
@@ -7,9 +7,23 @@
 function nestedForLoops (depth, width = depth) {
   let result = "\n";
 
+  let final = "";
```

Comment:
You don't really need this variable. You could do all the concatenation on the fly with just the result variable.